docs/codebase-notes/sources-reducer.md

```diff
@@ -14,13 +14,40 @@ interface Location {
   // other fields? sourceId?
 }
 
+interface SourceActor {
+  actor: string;
+  id: string;
+  introductionType?: unknown;
+  introductionUrl?: string;
+  source: string;
+  sourceMapURL?: string;
+  thread: string;
+  url: string;
+  breakableLines?: unknown | null;
+  breakpointPositions?: Map<unknown, unknown>
+}
+
+interface Source {
+  content: string | null;
+  extensionName?: string;
+  id: string;
+  introductionType?: unknown;
+  introductionUrl?: string;
+  isBlackBoxed: boolean;
+  isExtension: boolean;
+  isOriginal: boolean;
+  isPrettyPrinted: boolean;
+  relativeUrl?: string;
+  url?: string;
+}
+
 // Several types TBD here
```

Comment:
This seems like it probably isn't adding much, how would you feel about removing it now that you've got us to a full-TS place?

Reply:
Yeah, that's left over from the start of my notes on what the types looked like. I'll remove that in tomorrow's work on the actions files.